src/components/modules/toolbar/inline.ts

```diff
@@ -393,11 +441,26 @@ export default class InlineToolbar extends Module {
 
     /**
      * Filter buttons if Block Tool pass config like inlineToolbar=['link']
+     * Else filter them according to the default inlineToolbar property.
      */
+
+    let inlineToolbar;
+
     if (Array.isArray(inlineToolbarSettings)) {
-      buttons.forEach((button) => {
-        button.hidden = !inlineToolbarSettings.includes(button.dataset.tool);
-      });
+      inlineToolbar = inlineToolbarSettings;
+    } else if (Array.isArray(this.config.inlineToolbar)) {
+      inlineToolbar = this.config.inlineToolbar;
+    }
+
+    buttons.forEach((button) => {
+      button.hidden = !inlineToolbar.includes(button.dataset.tool);
```

Comment:
This line will throw an error if `inlineToolbarSettings` or `config.inlineToolbat` is not defined or defined but not an array.

Reply:
check latest commit